Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -115,7 +116,8 @@ vtkMRMLSliceNode::vtkMRMLSliceNode()
   this->LayoutColor[1] = vtkMRMLSliceNode::grayColor()[1];
   this->LayoutColor[2] = vtkMRMLSliceNode::grayColor()[2];
 
-  this->SetOrientationToAxial();
+  this->SetOrientationString("Reformat");
+  this->SetOrientationReference("Reformat");
```

Comment:
Now the function has been deprecated, may we should call it directly from `SetOrientationString` ?